test/e2e/cmd/cmd.go

```diff
@@ -137,3 +138,19 @@ func newAVSDeployCmd(def *app.Definition) *cobra.Command {
 
 	return cmd
 }
+
+func newCreate3DeployCmd(def *app.Definition) *cobra.Command {
+	cfg := app.DefaultCreate3DeployConfig()
```

Comment:
maybe we change our naming from: `avs-deploy` and `create3-deploy` => `deploy-avs` and `deploy-create3`

Since `create3-deploy` sounds like we're deploying a contract using create3, rather than actually deploying create3 itself

Reply:
Yeah had this discussion with pavel today. I'm more online for starting with `deploy`. Had thought of deploy subcommands `deploy create3`, `deploy avs` - the reason for `avs-deploy` and `create3-deploy` was to reduce risk of user error (i.e someone just deploying everything by typing deploy first and hitting enter)

Reply:
I actually think we might subcommand on contract type. That way we could add other things

```
avs deploy
avs updgrade
avs list-operators
create3 deploy
create3 get-address // just gets address without deployting
```